### PR TITLE
Update vent

### PR DESCRIPTION
**Pull request by tawhai**
Updates to the organisation of the main ventilation subroutine (and its dependencies). New organisation is needed so that a 'flow step' can be executed without having to run the entire ventilation solve. This is used for inert gas mixing.

### CODE REVIEW

File: src/lib/ventilation.f90 [hunk 1]
```diff
@@ -726,7 +791,8 @@ subroutine estimate_flow(dpmus,dt,err_est)
     enddo !nunit
 
     ! the estimate of error for the iterative solution
-    err_est=err_est/(flow_sum*DBLE(num_units))
+    if(abs(flow_sum*dble(num_units)).gt.zero_tol) &
+         err_est = err_est/(flow_sum*dble(num_units))
```
Comment (alysclark) [inline src/lib/ventilation.f90]: I get this is avoiding a divide by zero error  - is it the right thing to leave as is if sum of all flows is essentially zero?

**Review by alysclark — COMMENTED** [review]
Just one query on the change regarding errors in ventilation model. I can't think of an alternative so suspect this is the right thing to do, but just want to double check what you think...

**Comment by tawhai** [comment]
I had a warning message being output. But for a breath-hold we expect zero flow !

**Comment by alysclark** [comment]
Should there be the following (or similar) below that line:

else
    err_est = err_est/dble(num_units)

**Comment by tawhai** [comment]
OK – that sounds reasonable. I’ll make that change.

**Pull request merged**